### ISSUE TITLE
Link to bugzilla is broken

### ISSUE THREAD

**Issue by kwiqniss** [MicrosoftDocs/xamarin-docs#654]
the aka.ms link resolves to Bugzilla.xamarin.com, which has been retired. user redirected to visualstudio developer community site. 
The link from this blog https://blog.xamarin.com/xamarin-live-player-faq/ is also pointing to the same aka.ms.
What's the correct way to file a work item now?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6f641830-151c-230d-cdf6-da7b6962ad04
* Version Independent ID: 3b0e6ac7-c893-f087-7e73-cfe7d72ca5cd
* Content: [Limitations of Xamarin Live Player - Xamarin](https://docs.microsoft.com/en-us/xamarin/tools/live-player/limitations?tabs=android)
* Content Source: [docs/tools/live-player/limitations.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/tools/live-player/limitations.md)
* Product: **xamarin**
* GitHub Login: @topgenorth
* Microsoft Alias: **toopge**

**Comment by brendanzagaeski** [comment]
I made a quick adjustment on Bugzilla to help with this until the docs and aka.ms link are updated. I changed the bug entry link on Bugzilla <https://bugzilla.xamarin.com/enter_bug.cgi?product=Live%20Player> so that it no longer redirects straight to Developer Community but instead to the top-level bug entry page on Bugzilla. That top-level page includes a short description about how users should now file issues for Live Player via Report a Problem in Visual Studio.
